.github/workflows/dependency-review.yml

```
@@ -0,0 +1,30 @@
+        uses: actions/dependency-review-action@v4
+        with:
+          # Fail the action if vulnerabilities are found
+          fail-on-severity: moderate
+          # Allow licenses (customize as needed)
+          allow-licenses: MIT, Apache-2.0, BSD-2-Clause, BSD-3-Clause, ISC, 0BSD
+          # Deny licenses (customize as needed)
+          deny-licenses: GPL-3.0, AGPL-3.0
+          # Comment on PR with results
+          comment-summary-in-pr: always
+          # Warn about deprecated packages
+          warn-on-deprecated: true
```

actionlint (1.7.8)

input "warn-on-deprecated" is not defined in action "actions/dependency-review-action@v4". available inputs are "allow-dependencies-licenses", "allow-ghsas", "allow-licenses", "base-ref", "comment-summary-in-pr", "config-file", "deny-groups", "deny-licenses", "deny-packages", "external-repo-token", "fail-on-scopes", "fail-on-severity", "head-ref", "license-check", "repo-token", "retry-on-snapshot-warnings", "retry-on-snapshot-warnings-timeout", "show-openssf-scorecard", "vulnerability-check", "warn-on-openssf-scorecard-level", "warn-only"

(action)